src/lib/develco.ts

```diff
@@ -0,0 +1,159 @@
+                develcoPrimaryHwVersion: {ID: 0x8020, type: Zcl.DataType.OCTET_STR, manufacturerCode: Zcl.ManufacturerCode.DEVELCO},
+                develcoLedControl: {ID: 0x8100, type: Zcl.DataType.BITMAP8, manufacturerCode: Zcl.ManufacturerCode.DEVELCO},
+            },
+            commands: {},
+            commandsResponse: {},
+        }),
+    addCustomClusterManuSpecificDevelcoAirQuality: () =>
+        deviceAddCustomCluster('manuSpecificDevelcoAirQuality', {
+            ID: 0xfc03,
+            manufacturerCode: Zcl.ManufacturerCode.DEVELCO,
+            attributes: {
+                measuredValue: {ID: 0x0000, type: Zcl.DataType.UINT16},
+                minMeasuredValue: {ID: 0x0001, type: Zcl.DataType.UINT16},
+                maxMeasuredValue: {ID: 0x0002, type: Zcl.DataType.UINT16},
+                resolution: {ID: 0x0003, type: Zcl.DataType.UINT16},
+            },
+            commands: {},
+            commandsResponse: {},
+        }),
+    readGenBasicPrimaryVersions: (): ModernExtend => {
+        /*
+         * Develco (and there B2C brand Frient) do not use swBuildId
+         *  The versions are stored in develcoPrimarySwVersion and develcoPrimaryHwVersion, we read them during configure.
+         */
+        const configure: Configure[] = [
+            async (device, coordinatorEndpoint, definition) => {
+                for (const ep of device.endpoints) {
+                    if (ep.supportsInputCluster('genBasic')) {
+                        try {
+                            const data = await ep.read('genBasic', ['develcoPrimarySwVersion', 'develcoPrimaryHwVersion'], manufacturerOptions);
+
+                            if (data.hasOwnProperty('develcoPrimarySwVersion')) {
+                                device.softwareBuildID = data.develcoPrimarySwVersion.join('.');
+                            }
+
+                            if (data.hasOwnProperty('develcoPrimaryHwVersion')) {
+                                device.hardwareVersion = data.develcoPrimaryHwVersion.join('.');
+                            }
+
+                            device.save();
+                        } catch (error) {
+                            /* catch timeouts of sleeping devices */
+                        }
+                        break;
+                    }
+                }
+            },
+        ];
+        return {configure, isModernExtend: true};
+    },
+    voc: (args?: Partial<NumericArgs>) =>
+        numeric({
+            name: 'voc',
+            cluster: 'manuSpecificDevelcoAirQuality',
+            attribute: 'measuredValue',
+            reporting: {min: '1_MINUTE', max: '1_HOUR', change: 10},
+            description: 'Measured VOC value',
+            // from Sensirion_Gas_Sensors_SGP3x_TVOC_Concept.pdf
+            // "The mean molar mass of this mixture is 110 g/mol and hence,
+            // 1 ppb TVOC corresponds to 4.5 μg/m3."
+            scale: 4.5,
```

Comment:
@Koenkk klopt dit?

https://github.com/Koenkk/zigbee-herdsman-converters/pull/7706/files#diff-97dc2d48d7db27dd10bb142fd276583a4dfdff437808452d813cfb4bb65b2b59L161 het zou dit moeten doen dus `* 4.5` maar het lijkt eerder een `/ 4.5` te doen?

Reply:
Scale doet een `/`: https://github.com/Koenkk/zigbee-herdsman-converters/blob/a60047c170b562be05888513a9c493143ad91d7f/src/lib/modernExtend.ts#L1869

Reply:
Found a ScaleFunction did the trick nicely.